SPECS/check-restart/check-restart.sh

```diff
@@ -0,0 +1,6 @@
+OUTPUT=$(needs-restarting -r)
```

Comment:
Curious: which package provides the `needs-restarting` command? We should add it to the `Requires` for the new package.

Reply:
It needs dnf-utils. Added to Requires section.